pkg/asset/cluster/cluster.go

```diff
@@ -87,11 +87,13 @@ func (c *Cluster) Generate(parents asset.Parents) (err error) {
 
 	switch {
 	case installConfig.Config.Platform.AWS != nil:
+		identifier := map[string]string{
+			"tectonicClusterID": installConfig.Config.ClusterID,
+		}
+		identifier[fmt.Sprintf("kubernetes.io/cluster/%s", installConfig.Config.ObjectMeta.Name)] = "owned"
```

Comment:
any reason why no do it in the initialize above?

Reply:
> any reason why no do it in the initialize above?

Because I sidn't realize you could use function calls in map-literal keys ;).  Will fix.

Reply:
> any reason why no do it in the initialize above?

Done in f69e3fd -> 1412168.